.rubocop_todo.yml

```diff
@@ -1,18 +1,32 @@
-# Turn on RSpec cops
-require: rubocop-rspec
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2016-08-30 08:01:37 -0500 using RuboCop version 0.42.0.
+# on 2017-01-16 10:48:22 -0500 using RuboCop version 0.47.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-RSpec/SubjectStub:
+# Offense count: 8
+# Cop supports --auto-correct.
+# Configuration parameters: Include.
+# Include: **/Gemfile, **/gems.rb
+Bundler/OrderedGems:
   Exclude:
-    - 'spec/models/solr_endpoint_spec.rb'
+    - 'Gemfile'
+
+# Offense count: 56
+# Configuration parameters: CountComments, ExcludedMethods.
+Metrics/BlockLength:
+  Max: 220
```

Comment:
This seems really big, possibly it's an outlier (config block maybe?) Do we want to exclude just a few files and reduce the max?